src/Administrator.kt

```diff
@@ -1,40 +1,66 @@
-import java.util.* /** TODO: neeed to remove this because is not necessary */
+import User.Companion.EXIT_OPTION
 
 class Administrator(override val name: String, override val address: String, override val number: Int, override val dateOfBirth: Long, override val dateStarted: Long):User{
+    val DELETE_MOVIE_OPTION = "1"
+    val REGISTER_MOVIE_OPTION = "2"
+    val REGISTER_COPIES_OPTION = "3"
+    val REGISTER_ASSOCIATE_OPTION = "4"
+    val REGISTER_SALESMAN_OPTION = "5"
+
```

Comment:
Why are you using constant and are not using Enums ? 
I think these constants or enums could be better inside a specific file like Constants or Utils or something like that
What do you think ?